examples/quantization/brevitas/README.md

```diff
@@ -0,0 +1,39 @@
+# Example of quantizing decoder class LLMs with Brevitas from Transformers checkpoints
+
+The example shows an example on how to quantize a decoder-class LLM model through Brevitas:
+
+- Definition and instantiation of a parametrizable `BrevitasQuantizer`.
+- Optional conversion of the LLM model to an FX representation, leveraging Hugging Face transformers' tracer.
+- Support of executing post-training quantization (PTQ) algorithms and validation (SmoothQuant, GPTQ), while leveraging CPU offload from Hugging Face accelerate.
+- Validation of the quantized model.
+- Export of the quantized model as ONNX, QDQ-style.
+
+## Prerequisites
+
+The examples were tested using:
+- `brevitas>=0.10.2`
+- `torch>=2.1.2`
+- `transformers` installed from main (`pip install git+https://github.com/huggingface/transformers.git@4b236aed7618d90546cd2e8797dab5b4a24c5dce`)
+- `optimum>=1.17.0`
+- `optimum-amd`` installed from main (`pip install git+https://github.com/huggingface/optimum-amd`)
+- Optionally, `accelerate` installed from main (`pip install git+https://github.com/huggingface/accelerate.git`)
```

Comment:
Technically, at the moment installing accelerate is required, but using it is optional. I can fix this if necessary @fxmarty - let me know!